lib/go/go.mod

```diff
@@ -0,0 +1,5 @@
+module cli
```

Comment:
this should probably be something like `api-client`

the path must also reflect that `lib/go/api-client` and we need to build the lib with bazel (so we need a BUILD file)